README.md

```diff
@@ -1,2 +1,108 @@
-# template-php
-PHP template to start your Lenra app.
+<div id="top"></div>
+<!--
+*** This README was created with https://github.com/othneildrew/Best-README-Template
+-->
+
+
+
+<!-- PROJECT SHIELDS -->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+
+<h3 align="center">Template PHP</h3>
+
+  <p align="center">
+    This template provides just enough to get started with your PHP application.
+    <br />
+    <br />
+    <a href="https://github.com/lenra-io/template-php/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/lenra-io/template-php/issues">Request Feature</a>
+  </p>
+</div>
+
+
+
+
+<!-- GETTING STARTED -->
+
+## Prerequisites
+
+To properly run this template, you will have to make sure that the Lenra CLI and docker with docker-compose are installed on your computer.
+Installation instructions can be found here https://github.com/lenra-io/lenra_cli.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+To run the Rust template you first need to build the application.
+```console
+lenra build
+```
+
+Then you can start it.
+```console
+lenra start
+```
```

Comment:
Maybe replace the `build` and `start` by the `dev` command ?

```suggestion
To run the PHP template you can use the `dev` command that will build and start your app and enter in a custom terminal:

```console
lenra dev
```
```